server.go

```diff
@@ -181,6 +181,12 @@ func (server *Server) defaultConfigurationValues(configuration *types.Configurat
 			frontend.EntryPoints = server.globalConfiguration.DefaultEntryPoints
 		}
 	}
+	for _, backend := range configuration.Backends {
+		_, err := types.NewLoadBalancerMethod(backend.LoadBalancer)
+		if err != nil {
+			backend.LoadBalancer = &types.LoadBalancer{Method: "wrr"}
```

Comment:
Should this be logged ? :angel: (like "wrong backend.LoadBlancer, using default..")